backend/routes/statDictionaries.js

```diff
@@ -0,0 +1,59 @@
+let STAT_MEANS = {
```

Comment:
These should be `const` right? Or are you somehow changing the value after the export?

Reply:
Good point. I didn't know that exporting makes the pointer const. I have been changing it by accessing individual fields, so I can still keep my original logic. Changing to const in next pr.